gobblin-modules/gobblin-kafka-09/src/test/java/org/apache/gobblin/runtime/KafkaAvroJobStatusMonitorTest.java

```diff
@@ -130,78 +130,49 @@ public void testProcessMessageForSuccessfulFlow() throws IOException, Reflective
     } catch(InterruptedException ex) {
       Thread.currentThread().interrupt();
     }
-
-    Config config = ConfigFactory.empty().withValue(ConfigurationKeys.KAFKA_BROKERS, ConfigValueFactory.fromAnyRef("localhost:0000"))
-        .withValue(Kafka09ConsumerClient.GOBBLIN_CONFIG_VALUE_DESERIALIZER_CLASS_KEY, ConfigValueFactory.fromAnyRef("org.apache.kafka.common.serialization.ByteArrayDeserializer"))
-        .withValue(ConfigurationKeys.STATE_STORE_ROOT_DIR_KEY, ConfigValueFactory.fromAnyRef(stateStoreDir))
-        .withValue("zookeeper.connect", ConfigValueFactory.fromAnyRef("localhost:2121"));
-    MockKafkaAvroJobStatusMonitor jobStatusMonitor =  new MockKafkaAvroJobStatusMonitor("test",config, 1);
+    MockKafkaAvroJobStatusMonitor jobStatusMonitor = createMockKAJSM(new AtomicBoolean(false), ConfigFactory.empty());
```

Comment:
minor... and sorry, I abbreviated in a suggestion, but actually prefer writing it out as `createMockKafkaAvroJobStatusMonitor`, for clarity's sake